client/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,54 @@
+.blockquote {
+  @include font('HNM2');
+  text-align: center;
+
+  &--wb {
+    @include font('WB5');
+  }
+
+  &--lr {
+    @include font('LR2');
+    text-align: left;
+  }
+
+  footer {
```

Comment:
with the parsing we can do what ever we think is best practice - so 100% yes to adding styles to the children.

I'll get on that today.